packages/views/screens/managers/settings/settings.js

```diff
@@ -62,7 +62,7 @@ class Settings extends React.Component {
             <section>
               <h2>{t('Currency')}</h2>
               <div className={styles.fieldGroup}>
-                <CurrencySelector />
+                <CurrencySelector t={t} />
```

Comment:
dont pass translation from the parent
use translation hook inside the component 
```const {t} = useTranslation()```

Reply:
I only passed it as props this time since the same `const {t} = useTranslation()` was already used in the file, so I didn't want to reimport that in an immediate child component

Reply:
import it as many as you need 
its an important component of working independently without passing from parents